### PR TITLE
MODUSERBL-202 clean up permissions

### PR DESCRIPTION
**Pull request by NikitaSedyx**

### CODE REVIEW

File: descriptors/ModuleDescriptor-template.json [hunk 1]
```diff
@@ -124,7 +121,7 @@
         {
           "methods": ["GET"],
           "pathPattern": "/bl-users/by-username/{name}/open-transactions",
-          "permissionsRequired": ["users-bl.transactions.get"],
+          "permissionsRequired": ["users-bl.transactions-by-username.get"],
```
Comment (gurleenkaurbp) [inline descriptors/ModuleDescriptor-template.json:124]: Permission is not defined in module descriptor

Comment (gurleenkaurbp) [reply descriptors/ModuleDescriptor-template.json:124]: Also this permission is not added users-bl.all

**Review by gurleenkaurbp — COMMENTED** [review]
Has this case taken care ? 
{
[ERROR]   ""key"" : ""Permission is not defined in module descriptor"",
[ERROR]   ""value"" : ""perms.users.get""
[ERROR] }

Pom.xml changes are also to be added for the plugin which validates the MD file.

File: descriptors/ModuleDescriptor-template.json (hunk 1 above)
Comment (NikitaSedyx) [reply descriptors/ModuleDescriptor-template.json:124]: oh, yep, I forgot to rename new one, it's actually added

**Comment by NikitaSedyx** [comment]
> Has this case taken care ?
{
[ERROR] ""key"" : ""Permission is not defined in module descriptor"",
[ERROR] ""value"" : ""perms.users.get""
[ERROR] }

it was removed from permissions required as endpoint can't depend from other module permission

**Comment by NikitaSedyx** [comment]
@gurleenkaurbp done

**Review by gurleenkaurbp — APPROVED** [review]

**Review by Vignesh-kalyanasundaram — APPROVED** [review]

**Review by SerhiiNosko — APPROVED** [review]

**Pull request merged**